spec/unit/provider/user/useradd_spec.rb

```diff
@@ -144,6 +144,16 @@
         provider.expects(:execute).with(all_of(includes('/usr/sbin/lusermod'), includes('-e')))
         provider.create
       end
+
+      it 'should set password age rules locally with forcelocal = true' do
```

Comment:
The describe for this block says ..."and forcelocal=true", so ...''with forcelocal = true'' might be redundant here.

Reply:
yeah, I'll fix that